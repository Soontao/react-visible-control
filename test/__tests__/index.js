import React from 'react';
import renderer from "react-test-renderer";
import Test from '../Test';

test('render test', () => {
  const component = renderer.create(
    <Test />
  )
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})