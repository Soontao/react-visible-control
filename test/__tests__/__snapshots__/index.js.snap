// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render test 1`] = `
<div>
  <p>
    home
  </p>
  <p>
    page1
  </p>
  <p>
    you cant access page2, but you could show a failback thing here
  </p>
</div>
`;
